WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -243,6 +243,20 @@ class NotificationsViewController: UITableViewController, UIViewControllerRestor
         }
     }
 
+    override func applicationFinishedRestoringState() {
+        super.applicationFinishedRestoringState()
+
+        guard let navigationControllers = navigationController?.children else {
+            return
+        }
+
+        for case let detailVC as NotificationDetailsViewController in navigationControllers {
+            if detailVC.onDeletionRequestCallback == nil, let note = detailVC.note {
+                configureDetailsViewController(detailVC, withNote: note)
```

Comment:
Looks **definitely** way better than the fatalError I've wired in the DetailsViewController!!

(Thank you for getting this patched up Nate!)

Reply:
WHY HELLO THERE 👋 

Thanks for the show of support @jleandroperez 😀

Reply:
Hope you're doing great over there Nate! (I got the notification on the master issue!!) 😄